src/EnergyPlus/SetPointManager.cc

```diff
@@ -134,6 +138,26 @@ using namespace CurveManager;
 using Psychrometrics::PsyCpAirFnW;
 using Psychrometrics::PsyHFnTdbW;
 
+constexpr std::array<std::string_view, static_cast<int>(CtrlVarType::Num)> controlTypeName = {"Temperature",
+                                                                                              "MaximumTemperature",
+                                                                                              "MinimumTemperature",
+                                                                                              "HumidityRatio",
+                                                                                              "MaximumHumidityRatio",
+                                                                                              "MinimumHumidityRatio",
+                                                                                              "MassFlowRate",
+                                                                                              "MaximumMassFlowRate",
+                                                                                              "MinimumMassFlowRate"};
+
+constexpr std::array<std::string_view, static_cast<int>(CtrlVarType::Num)> controlTypeNameUC = {"TEMPERATURE",
+                                                                                                "MAXIMUMTEMPERATURE",
+                                                                                                "MINIMUMTEMPERATURE",
+                                                                                                "HUMIDITYRATIO",
+                                                                                                "MAXIMUMHUMIDITYRATIO",
+                                                                                                "MINIMUMHUMIDITYRATIO",
+                                                                                                "MASSFLOWRATE",
+                                                                                                "MAXIMUMMASSFLOWRATE",
+                                                                                                "MINIMUMMASSFLOWRATE"};
+
```

Comment:
Previously, we had a if-else statement for these but changed to arrays, recommended by Amir. Also, these arrays are moved from `.hh` file to `.cc` file recommended by Amir.